src/test/java/com/google/spanner/liquibase/LiquibaseTests.java

```diff
@@ -87,13 +93,20 @@ static TestHarness.Connection getSpannerReal() throws SQLException {
 
   // Return spannerEmulator instance
   // It is shared across tests.
-  private static TestHarness.Connection spannerEmulator;
+  public TestHarness.Connection getSpannerEmulator(Dialect dialect) throws SQLException {
+    switch (dialect) {
+      case POSTGRESQL:
+        if (spannerPgConnection == null) {
+          spannerPgConnection = TestHarness.useSpannerEmulator(dialect);
```

Comment:
For a follow-up PR: We don't need to create two emulator instances. We can use one emulator instance and create two different databases with different dialects.